packages/contentstack-seed/package.json

```diff
@@ -22,7 +22,7 @@
     "@types/node": "^14.14.32",
     "@types/tar": "^6.1.3",
     "@types/tmp": "^0.2.0",
-    "axios": "^1.6.4",
+    "axios": "^1.7.4",
```

Comment:
why did we bump the seed here, not the bulk publish, version bump can be done once we create PR  to development branch

Reply:
@shafeeqd959   SCA scan workflow is failing due to axios package bump so, i have upgraded the axios version